### PR TITLE
Adding guidance for test jobs in Sig Node

### PR DESCRIPTION
**Pull request by ndixita**
/assign @SergeyKanzhelev

### CODE REVIEW

File: config/jobs/kubernetes/sig-node/README.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+   <td>N-3, N-2, N-1, N, main
+   </td>
+  </tr>
+</table>
+
+
+
+## Naming Convention
+
+{cri}-{cgroup}-{os}-{test-type}-release-{release-branch}
```
Comment (upodroid) [inline config/jobs/kubernetes/sig-node/README.md:44]:
```suggestion
{pull|ci}-kubernetes-e2e-node-{cri}-{cgroup}-{cloud}-{os}-{arch}-{test-type}-release-{release-branch}
```

File: config/jobs/kubernetes/sig-node/README.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+**Note:** Specify variable value in the name only if it isn’t set to default in a test tab.
+
+
+## Test Tabs
+
+#### 1. **sig-node-default (i.e. rename sig-node-release-blocking)**
+
+<table>
```
Comment (upodroid) [inline config/jobs/kubernetes/sig-node/README.md:53]: We'll need a dashboard for:

- for each architecture
- for each cloud

**Review by upodroid — CHANGES_REQUESTED** [review]
I'm working on a prowjob generator in #29944 and will configure the matrix based on what we merge in this PR.

The generator is almost ready, I need to fix a few things in kubetest2

File: config/jobs/kubernetes/sig-node/README.md (hunk 2 above)
Comment (ndixita) [reply config/jobs/kubernetes/sig-node/README.md:53]: I will discuss this in tomorrow's sig CI meeting and accommodate this

**Comment by dims** [comment]
/approve

this is a great start. thanks @ndixita.

yes, +1 to automation to populate these @upodroid

**Comment by SergeyKanzhelev** [comment]
/lgtm
/approve

Let's merge and iterate

**Comment by upodroid** [comment]
+1

**Pull request merged**